src/main/scala/mesosphere/marathon/api/v2/validation/AppValidation.scala

```diff
@@ -304,7 +305,7 @@ trait AppValidation {
     }
   )
 
-  def validateCanonicalAppUpdateAPI(enabledFeatures: Set[String]): Validator[AppUpdate] = forAll(
+  def validateCanonicalAppUpdateAPI(enabledFeatures: Set[String], normalizationConfig: AppNormalization.Config): Validator[AppUpdate] = forAll(
```

Comment:
please use a `defaultNetworkName: Option[String]` here. let's not mix normalization APIs into the validation code